step-capstone/src/__tests__/TripTests.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Trip from '../components/Trip';
+import renderer from 'react-test-renderer';
+import {FirebaseContext } from '../components/Firebase';
+import firebaseMock from '../components/firebaseMock'
+
+test('Trip displays correctly', async () => {
+  sessionStorage.setItem("userId", "testUser");
+  sessionStorage.setItem("tripId", "testTrip");
+  
+  const tripComponent = renderer.create(
+    <FirebaseContext.Provider value={new firebaseMock()}>
+      <Trip/>
+    </FirebaseContext.Provider>
+    
+  );
+  await new Promise(res => setTimeout(() => { res() }, 0.5))
```

Comment:
Nit: Please add a similar comment anyplace you had to put a wait, since that's kind of unusual for tests.

Reply:
done